ReleaseNotes.md

```diff
@@ -0,0 +1,168 @@
+# JustSaying 5.0.0 Release Notes
+This release has been a long time in the making, I want to say a huge thank you to everyone that contributed, and to everyone that was patient in waiting for this release.
+
+The big changes in this release are:
+
+### Re-implemented on top of AWSSDK v3
+In version 3 of the AWSSDK, Amazon changed the package names, and split them into smaller individual packages per service. They also removed synchronous calls for target frameworks other than the full .NET framework.
```

Comment:
Link to https://aws.amazon.com/about-aws/whats-new/2015/07/version-3-of-the-aws-sdk-for-net-is-available-for-production-use-in-net-3-5-and-4-5/